testSrc/com/intellij/plugins/haxe/ide/HaxeSemanticAnnotatorTest.java

```diff
@@ -154,4 +154,16 @@ public void testPropertiesSimpleCheck() throws Exception {
   public void testPropertyAllowNonConstantInitialization() throws Exception {
     doTestNoFixWithWarnings();
   }
+
+  public void testOverrideSignature() throws Exception {
+    doTest("Remove argument");
+  }
+
```

Comment:
How about a couple of tests to ensure that the comma detection and deleting in remove() works as expected?